media/libstagefright/Android.mk

```diff
@@ -207,6 +207,9 @@ LOCAL_CFLAGS += -Wno-multichar
 ifeq ($(BOARD_USE_SAMSUNG_COLORFORMAT), true)
 LOCAL_CFLAGS += -DUSE_SAMSUNG_COLORFORMAT
 
+ifeq ($(BOARD_IGNORE_INTRA_REFRESH_MODE_ERROR), true)
+LOCAL_CFLAGS += -DIGNORE_AVC_INTRA_MACROBLOCK_REFRESH_MODE_ERROR
```

Comment:
Maybe other platforms using the same Hardware encoder (OMX.SEC.AVC.Encoder) may also find this fix useful.
However, thank you for adding these missing files :+1: